lodash.js

```diff
@@ -14784,9 +14784,12 @@
       , 'g');
 
       // Use a sourceURL for easier debugging.
+      // The sourceURL gets injected into the source that's eval-ed, so be careful
+      // with lookup (in case of e.g. prototype pollution), and strip newlines if any.
+      // A newline wouldn't be a valid sourceURL anyway, and it'd enable code injection.
       var sourceURL = '//# sourceURL=' +
-        ('sourceURL' in options
-          ? options.sourceURL
+        (hasOwnProperty.call(options, 'sourceURL')
+          ? options.sourceURL.replace(/[\r\n]/g, ' ')
```

Comment:
Can you coerce `options.sourceURL` to a string. Something like `(options.sourceURL + '').replace` would do.

Reply:
Fixed :)